src/optionstr.c

```diff
@@ -3135,7 +3133,7 @@ expand_set_nrformats(optexpand_T *args, int *numMatches, char_u ***matches)
 #if defined(FEAT_EVAL) || defined(PROTO)
 /*
  * One of the '*expr' options is changed: 'balloonexpr', 'diffexpr',
- * 'findexpr', 'foldexpr', 'foldtext', 'formatexpr', 'includeexpr',
+ * 'findfunc', 'foldexpr', 'foldtext', 'formatexpr', 'includeexpr',
```

Comment:
`did_set_optexpr()` isn't called for 'findfunc', so 'findfunc' shouldn't be in the comment here.